packages/x-date-pickers-pro/src/DateRangePicker/DateRangePicker.tsx

```diff
@@ -108,78 +42,33 @@ export const DateRangePicker = React.forwardRef(function DateRangePicker<TDate>(
   useLicenseVerifier('x-date-pickers-pro', releaseInfo);
 
   const {
-    calendars = 2,
-    value,
-    onChange,
-    mask = '__/__/____',
-    startText = 'Start',
-    endText = 'End',
-    inputFormat: passedInputFormat,
-    minDate: minDateProp,
-    maxDate: maxDateProp,
+    cancelText,
+    desktopModeMediaQuery = '@media (pointer: fine)',
+    DialogProps,
+    okText,
+    PopperProps,
+    showTodayButton,
+    todayText,
+    TransitionComponent,
```

Comment:
Do you know why this prop is not transmitted to the rendered components event if it's a prop that can be used with `<DesktopDateRangePicker/>`?

Reply:
(added)